pyproject.toml

```diff
@@ -115,7 +115,7 @@ regex = ">=2020.6,<2022.11"
 joblib = ">=0.15.1,<1.3.0"
 sentry-sdk = ">=0.17.0,<1.15.0"
 aio-pika = ">=6.7.1,<9.0.0"
-aiogram = "<2.24"
+aiogram = "~2.25.1"
```

Comment:
Any reason why this constraint is not "<2.26" as suggested by dependabot?

Reply:
Good question. I always thought that `<` is more appropriate when we have some range of versions. As I see in our pyproject.toml, that's the case for most of our dependencies. If single version is used, tilda is slightly more readable, as you don't need to decrement version in the head (`~2.25.1` vs `<2.26`). And I think dependabot suggested `<2.26` just because before it was `<2.24` before.

But I don't have very strong opinion about this :).

Reply:
Yes, sure, will change for depbot suggested version